src/compiler/commandLineParser.ts

```diff
@@ -1115,7 +1115,8 @@ namespace ts {
         target: ScriptTarget.ES5,
         strict: true,
         esModuleInterop: true,
-        forceConsistentCasingInFileNames: true
+        forceConsistentCasingInFileNames: true,
+        skipLibCheck: true

```

Comment:
From the design meeting::
> Ask is for skipLibCheck to be included in tsc --init, not that it's turned on.

Reply:
Yeah I agree - it looks like that. I think the design notes were the first part of the discussion  - the JSON dump we worked on included it as true

![Screen Shot 2020-04-07 at 8 56 51 AM](https://user-images.githubusercontent.com/49038/78671861-d8c9ca80-78ad-11ea-87dd-7cf9e6898f2c.png)